views/admin_portal.pug

```diff
@@ -23,25 +23,27 @@ block content
                     a(href='/admin/users') Users
                   li
                     a(href='/admin/organizations') Developer Organizations
+                if user.roles & Constants.Role.BLOG_EDITOR
                   li
                     a(href='/admin/blog') Blog Archive
                 if WITH_THINGPEDIA === 'embedded'
                   if user.roles & Constants.Role.ADMIN
                     li
                       a(href='/admin/training') Natural Language Training
-                  li
-                    a(href='/admin/review-queue') Unreviewed Devices
-                  li
-                    a(href='/thingpedia/entities') Thingpedia Entities
-                  li
-                    a(href='/thingpedia/strings') Thingpedia String Datasets
-                  li
-                    a(href='/thingpedia/snapshots') Thingpedia Snapshots
-                  li
-                    a(href='/thingpedia/datasets') Thingpedia Datasets
-                if (user.roles & Constants.Role.ADMIN) && WITH_THINGPEDIA === 'embedded'
-                  li
-                    a(href='/mturk') MTurk Batches
+                  if user.developer_status >= 3
```

Comment:
I guess use symbolic constants here too?

Reply:
Yes, we need to do a pass for developer_status too. It's more complicated because we're using it in various places, but I guess it's fine if we forget to change one until we actually add new developer statuses?